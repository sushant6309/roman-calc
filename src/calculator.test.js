/**
 * Created by apple on 17/06/17.
 */
import React from 'react';
import renderer from 'react-test-renderer';
import Square from './square';
import Board from './board';
import Calculator from './calculator'
import { mount } from 'enzyme';

test('Testing Roman Calculator', () => {

  const calculator = mount(<Calculator/>);
  //calculator.handleClick('V');
  console.log(calculator);

});