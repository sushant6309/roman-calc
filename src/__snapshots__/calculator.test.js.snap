// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Testing Roman Calculator 1`] = `
<div
  className="container-display"
>
  <div
    className="result-container"
  >
    <div
      className="text-display"
    >
      
    </div>
    <div
      className="result-heading"
    >
      
    </div>
  </div>
  <div
    className="main-container1"
  >
    <div>
      <div
        className="calculator-container"
      >
        <button
          className="block-1"
          onClick={[Function]}
        >
          /
        </button>
        <button
          className="block-1"
          onClick={[Function]}
        >
          *
        </button>
        <button
          className="block-1"
          onClick={[Function]}
        >
          -
        </button>
        <button
          className="block-1"
          onClick={[Function]}
        >
          +
        </button>
      </div>
      <div
        className="calculator-container"
      >
        <button
          className="block-2"
          onClick={[Function]}
        >
          C
        </button>
        <button
          className="block-2"
          onClick={[Function]}
        >
          D
        </button>
        <button
          className="block-2"
          onClick={[Function]}
        >
          M
        </button>
        <button
          className="block-1"
          onClick={[Function]}
        >
          Clr
        </button>
      </div>
      <div
        className="calculator-container-4"
      >
        <button
          className="block-2"
          onClick={[Function]}
        >
          I
        </button>
        <button
          className="block-2"
          onClick={[Function]}
        >
          V
        </button>
        <button
          className="block-2"
          onClick={[Function]}
        >
          X
        </button>
        <button
          className="block-2"
          onClick={[Function]}
        >
          L
        </button>
      </div>
      <div
        className="calculator-container-4"
      >
        <button
          className="delete"
          onClick={[Function]}
        >
          Del
        </button>
        <button
          className="delete-2"
          onClick={[Function]}
        >
          =
        </button>
      </div>
    </div>
  </div>
</div>
`;

exports[`Testing Roman Calculator 2`] = `
<div>
  <div
    className="calculator-container"
  >
    <button
      className="block-1"
      onClick={[Function]}
    >
      /
    </button>
    <button
      className="block-1"
      onClick={[Function]}
    >
      *
    </button>
    <button
      className="block-1"
      onClick={[Function]}
    >
      -
    </button>
    <button
      className="block-1"
      onClick={[Function]}
    >
      +
    </button>
  </div>
  <div
    className="calculator-container"
  >
    <button
      className="block-2"
      onClick={[Function]}
    >
      C
    </button>
    <button
      className="block-2"
      onClick={[Function]}
    >
      D
    </button>
    <button
      className="block-2"
      onClick={[Function]}
    >
      M
    </button>
    <button
      className="block-1"
      onClick={[Function]}
    >
      Clr
    </button>
  </div>
  <div
    className="calculator-container-4"
  >
    <button
      className="block-2"
      onClick={[Function]}
    >
      I
    </button>
    <button
      className="block-2"
      onClick={[Function]}
    >
      V
    </button>
    <button
      className="block-2"
      onClick={[Function]}
    >
      X
    </button>
    <button
      className="block-2"
      onClick={[Function]}
    >
      L
    </button>
  </div>
  <div
    className="calculator-container-4"
  >
    <button
      className="delete"
      onClick={[Function]}
    >
      Del
    </button>
    <button
      className="delete-2"
      onClick={[Function]}
    >
      =
    </button>
  </div>
</div>
`;
